### PR TITLE
add new osx dmg background picture

### PR DESCRIPTION
**Pull request by jonasschnelli**

### CODE REVIEW

**Comment by jonasschnelli** [comment]
prebuilt binaries: https://builds.jonasschnelli.ch/pulls/5683/

File: contrib/macdeploy/fancy.plist [hunk 1]
```diff
@@ -10,7 +10,7 @@
 		<integer>620</integer>
 	</array>
 	<key>background_picture</key>
-	<string>background.png</string>
+	<string>background.tiff</string>
```
Comment (jonasschnelli) [inline contrib/macdeploy/fancy.plist]: tiff is used to include a retina and non-retina background (according to https://developer.apple.com/library/mac/documentation/GraphicsAnimation/Conceptual/HighResolutionOSX/Optimizing/Optimizing.html)

**Comment by theuni** [comment]
@jonasschnelli Please review/pull in the top 3 commits here: https://github.com/theuni/bitcoin/commits/osx-background-image . They'll fix OSX builds from Linux/Gitian after these changes.

These sync up the way that we deal with the background images. Prior to these changes, different paths were used depending whether the build was native OSX or Linux cross.

Since Linux can't mount the image and run the applescripts, we have to use a pre-generated DS_Store file to grab the proper layout. Also, Linux can't set the "invisible" property on the background image, so we work-around by moving it into a ".background" folder, which OSX hides by default.

Here's a cross-built dmg to verify that everything looks correct: https://bitcoincore.org/cfields/Bitcoin-Qt-82f9b72c.dmg

As for the changes themselves, IMO the text would look nicer centered. Also, the arrow looks a bit... cartooney.

**Comment by jonasschnelli** [comment]
@theuni 
Perfect. I reviewed the three commits (82f9b72, e9383dc, 439b352). Looks good to me. ACK with the nit/question of the output size factor.

Also updated the image once again to center the text. I was getting inspired by the firefox background image (there the "Firefox" text is also left aligned). But centered looks good IMO.

**Comment by jonasschnelli** [comment]
pulled in 82f9b72, e9383dc, 439b352

**Comment by jonasschnelli** [comment]
It looks more or less like this:
bildschirmfoto 2015-01-20 um 14 29 28

**Comment by jonasschnelli** [comment]
current screen:

bildschirmfoto 2015-01-20 um 15 22 28

**Comment by jonasschnelli** [comment]
binary to test: https://bitcoin.jonasschnelli.ch/pulls/5683/

**Comment by fanquake** [comment]
ACK

**Pull request merged**